README.md

```diff
@@ -73,7 +73,7 @@ Number of failures: 3
 CloudFormation Guard can be used to evaluate security best practices for infrastructure deployed via CloudFormation. A number of example rules are included:
 
 ```
-$> cfn-guard check -t Examples/security_template.json -r Examples/security_rules.ruleset
+$> cfn-guard check -t Examples/security-template.json -r Examples/security-template.ruleset --strict-checks
```

Comment:
Reviewing [`--strict-checks`](https://github.com/aws-cloudformation/cloudformation-guard/blob/master/cfn-guard/README.md#strict-checks) additions
```diff
$ diff <(cfn-guard check -t Examples/security-template.json -r Examples/security-template.ruleset) <(cfn-guard check -t Examples/security-template.json -r Examples/security-template.ruleset --strict-checks)
> [CloudTrailTrailFail] failed because CloudTrail logs should be encrypted with a CMK. See: https://docs.aws.amazon.com/awscloudtrail/latest/userguide/encrypting-cloudtrail-log-files-with-aws-kms.html
> [CodeBuildProject] failed because CodeBuild projects should be built in a VPC
> [CodeBuildProject] failed because CodeBuild should use a CMK instead of AWS-provided KMS keys
> [DMSInstance] failed because A KMS key should be provided for encryption
> [EKSCluster] failed because EKS cluster encryption config should be set
> [FsXFilesystem] failed because CMKs should be used instead of AWS-provided KMS keys
> [GuardDutyDetector] failed because Detector should be enabled
> [ImageBuilderComponent] failed because A KMS key should be provided for encryption
> [ImageBuilderInfrastructureConfiguration] failed because Logging should be enabled
> [LambdaFunction] failed because KMS key should be set for encryption
> [MSKCluster] failed because Clusters should authenticate clients with TLS
> [NeptuneCluster] failed because CloudWatch log exports should be enabled
> [OpsWorksApp] failed because TLS should be enabled
> [RDSDBCluster] failed because CloudWatch log exports should be enabled
> [RDSDBInstance] failed because CloudWatch log exports should be enabled
> [SNSTopic] failed because KMS encryption should be enabled
> [SQSQueue] failed because KMS encryption should be enabled
> [SageMakerNotebook] failed because KMS encryption should be enabled
> [SecretsManagerSecretFail] failed because Secrets should be encrypted using a CMK instead of AWS-provided KMS keys
> [StepFunctionsStateMachine] failed because Logging should be enabled
< Number of failures: 75
> Number of failures: 95
```

Reply:
hey @matthewdfuller - any luck on this? Happy to take it in whatever direction makes sense for this PR.